quilt/tools/build.py

```diff
@@ -23,15 +25,14 @@ def _is_internal_node(node):
     return all(isinstance(x, dict) for x in node.values())
 
 def _pythonize_name(name):
-    safename = re.sub('[^A-Za-z0-9_]+', '_', name)
-    starts_w_number = re.match('^[0-9].*', safename)
-    if starts_w_number:
-        safename = ("abc%s" % safename)
-
+    safename = re.sub('[^A-Za-z0-9]+', '_', name)
     safename = safename.strip('_')
```

Comment:
very minor. why not chain the above two calls? maybe that's not our codebase style just asking.